pkg/operation/terraformer/terraformer.go

```diff
@@ -398,7 +398,7 @@ func (t *Terraformer) podSpec(scriptName string) *corev1.PodSpec {
 					},
 					Limits: corev1.ResourceList{
 						corev1.ResourceCPU:    resource.MustParse("200m"),
```

Comment:
Should we even set higher limits, just to be safe? Also, I'd increase the requests. WDYT about

```yaml
resources:
  requests:
    cpu: 200m
    memory: 512Mi
  limits:
    cpu: 500m
    memory: 2Gi
```

Even if in some cases this is over-provisioned it's not too bad because these Terraform pods run anyways only once a day.

Reply:
Increases the memory limit to `1.5Gi`.